modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -278,22 +278,25 @@ export class Keychains implements IKeychains {
    * @return {Promise<KeychainsTriplet>} newly created User, Backup, and BitGo keys
    */
   async createMpc(params: CreateMpcOptions): Promise<KeychainsTriplet> {
-    if (_.isUndefined(params.passphrase)) {
-      throw new Error('missing required param passphrase');
-    }
-
     let MpcUtils;
     switch (params.multisigType) {
       case 'tss':
         MpcUtils = this.baseCoin.getMPCAlgorithm() === 'ecdsa' ? ECDSAUtils.EcdsaUtils : EDDSAUtils.default;
         break;
       case 'blsdkg':
+        if (_.isUndefined(params.passphrase)) {
```

Comment:
Shouldn't we also make the passphrase optional for blsdkg?

Reply:
Per scope of the ticket, let's get this in and create a follow up ticket to address Leo's question.